nemo_curator/modules/semantic_dedup/embeddings.py

```diff
@@ -114,7 +114,8 @@ def __init__(
         self,
         embedding_model_name_or_path: str,
         embedding_batch_size: int,
-        embedding_output_dir: str,
+        cache_dir: Optional[str] = None,
+        embeddings_save_loc: str = "embeddings",
```

Comment:
We deprecate `embedding_output_dir` in favor of `cache_dir` and `embeddings_save_loc` for `EmbeddingCreator`, which matches the logic in the `SemDedup` class.